README.md

```diff
@@ -16,8 +16,8 @@ i.e. sample load, CCS case, Individual Response.
 When creating or updating a case this application will emit msgs to be consumed by RH, Field & the 
 action-scheduler as appropriate.  Action-scheduler has a subset copy of the case data (CCS cases are excluded)
 
-CCS cases are a special cases which are never passed to the action-scheduler or RH.  They are emitted 
-to field if Listed without being supplied with a qid, unaddressed or refusal.
+CCS cases are special cases which are never passed to the action-scheduler and RH. They are emitted 
+to field if Listed without being supplied with a QID, unaddressed or refusal. CCS case event messages (such as 'case updated') are still emitted to action-scheduler but will be ignored on arrival.
```

Comment:
I don't think this is true now

Reply:
They'll be ignored on arrival as a result of the change from [this PR](https://github.com/ONSdigital/census-rm-action-scheduler/pull/59/files#diff-7b6da221a89d3c959fcae48e8e87a341R45) I think?